ui/dashboards/src/views/ViewDashboard/ViewDashboard.tsx

```diff
@@ -47,11 +47,16 @@ export function ViewDashboard(props: ViewDashboardProps) {
   const { spec } = dashboardResource;
   const dashboardDuration = spec.duration ?? '1h';
   const initialTimeRange = useInitialTimeRange(dashboardDuration);
+  const initialRefreshInterval = useInitialRefreshInterval();
```

Comment:
Similar to `spec.duration` we'll need something like `spec.refresh` added to our [schema](https://github.com/perses/perses/blob/main/docs/dashboard.md#spec) so individual dashboards can customize this. Would be nice for it to be optional though unlike [duration](https://github.com/perses/perses/blob/70bcb3856bc4b53bc3a37abb31d23fcb64ec5aad/ui/core/src/model/dashboard.ts#L28-L35)